docs/content/guides/migrations-and-breaking-changes.md

```diff
@@ -1,55 +0,0 @@
-# Migrations and Notable Changes
```

Comment:
If you remove this file then you should also remove it from the navigation in `mkdocs.yml`